models/attribute-group/src/transformers.ts

```diff
@@ -0,0 +1,24 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TAttributeGroup, TAttributeGroupGraphql } from './types';
+const transformers = {
+  graphql: Transformer<TAttributeGroup, TAttributeGroupGraphql>('graphql', {
+    buildFields: ['createdBy', 'lastModifiedBy'],
+    addFields: ({ fields }) => ({
+      __typename: 'AttributeGroup',
+      nameAllLocales: [
+        {
+          locale: 'en',
+          value: fields.name,
```

Comment:
`name` and `description` should be `LocalizedString` types so we would need to use that model helpers (`toLocalizedField`) in order to transform them into `LocalizedFields`.
We also need those properties to be buildable.

Reply:
Done in https://github.com/commercetools/test-data/pull/652/commits/ac5cfe5c389dddd194f3880105d4b470b2da6899